prodtests/full-system-test/dpl-workflow.sh

```diff
@@ -205,6 +209,14 @@ done
 PVERTEX_CONFIG="--vertexing-sources $TRACK_SOURCES --vertex-track-matching-sources $TRACK_SOURCES"
 has_detector_reco FT0 && PVERTEX_CONFIG+=" --validate-with-ft0"
 
+# In case matching of TPC-TRD tracks is requested, force off the TRD filtering
+TRD_SOURCES_ARRAY=$(echo $TRD_SOURCES | tr "," "\n")
+for src in $TRD_SOURCES_ARRAY; do
+  if [[ $src == "TPC-TRD" ]]; then
+    TRD_FILTER=""
+  fi
+done
+
```

Comment:
This 1-liner should do the same :)
```
[[ $TRD_SOURCES =~ (^|,)"TPC-TRD"(,|$) ]] && TRD_FILTER=""
```
For my understanding, why do you want to disable the filtering in case we match to TPC? As far as I understand @shahor02 , this is mostly for constraining the TPC tracks before the TPC-ITS matching. So if we do not do TPC-ITS matching for that collision, I don't know if we need the TRD matching anyway?

Reply:
indeed, I don't see any reason to run TPC-TRD matching in sync stage, unless for QC